src/lightning_app/utilities/cluster_logs.py

```diff
@@ -26,6 +26,7 @@ class _ClusterLogEventLabels:
     issuer: Optional[str] = None
     error: Optional[str] = None
     errorVerbose: Optional[str] = None
```

Comment:
nit: can this be changed to `error_verbose`? not necessarily for this PR

Reply:
Sure, will do in the next portion of improvements!